app/src/main/res/layout/title_bar.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Mała uwaga - tag merge nie działa z bindingiem, który może być przydatny dla customowych widoków (np. wiele rzeczy, w tym click handlery, można przekazać jako customowa properta w xml).

Oczywiście nie wymagam, żebyś to przepisywał. Jak będzie okazja to pokażę Ci o czym mowa :)

Reply:
a widzisz, ja już przyzwyczajony do projketu bez bindingu i wyprowawadziłem metody by potem programistycznie ustawiac parametry i event'y. Zrobimy tak, że zrobie na razie issue by to refactorowac. Jak aktualnie nie bedzie nic do zrobienia to sie tym zajme, bo pare mam tutaj niewiadomych ;)